src/fixes/quote_character.rs

```diff
@@ -13,7 +13,9 @@ impl Fix for QuoteCharacterFixer {
         let value = line.get_value()?;
         let pure_val = value.replace('\'', "").replace('\"', "");
 
-        line.raw_string = format!("{}={}", line.get_key()?, pure_val);
+        if !value.contains('#') {
```

Comment:
As I can see, these changes related only to `Fixer`. What about `Checker`? 🤔 

I'd suggest to move this condition there. Because it isn't really responsibility of fixer 🤔

I mean `Fixer` would apply changes only if `Checker` found any warnings/issues. And there is no sense to double check it. 
Fixer is quite dumb.

Reply:
I completely missed this when I looked for relevant tests. I will amend my proposed change.

Reply:
I hope that my altered change is satisfactory.